yarn.lock

```diff
@@ -2653,47 +2653,54 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@patternfly/patternfly@npm:6.2.0-prerelease.28, @patternfly/patternfly@npm:^6.2.0-prerelease.28":
+"@patternfly/patternfly@npm:6.2.0-prerelease.28":
   version: 6.2.0-prerelease.28
   resolution: "@patternfly/patternfly@npm:6.2.0-prerelease.28"
   checksum: 10c0/36a9acb6973471deb741f67922b13baf38258e5a1b400d67ec87caf76e48000b1188216d74f9bc5f348d6b33f7373858adb7ac8dd62887eb4d75b772abdab3db
   languageName: node
   linkType: hard
 
-"@patternfly/quickstarts@npm:6.2.0-prerelease.8":
-  version: 6.2.0-prerelease.8
-  resolution: "@patternfly/quickstarts@npm:6.2.0-prerelease.8"
+"@patternfly/patternfly@npm:^6.2.0":
+  version: 6.2.0
+  resolution: "@patternfly/patternfly@npm:6.2.0"
+  checksum: 10c0/cb316af5cdbdd71ee4a46a562b0f21eb8c4f73a00e9cfaef451bec2ae695d16ce696b9650471d460c627640ec080a52b4e3c01053cd43cc63e2449f3a0a49857
+  languageName: node
+  linkType: hard
+
+"@patternfly/quickstarts@npm:6.1.0":
```

Comment:
Why is 6.1 quickstarts still in the lock file?

Reply:
all set i didn't push the lock file.